Source/Rive/Public/Rive/RiveTextureObject.h

```diff
@@ -77,6 +77,7 @@ class RIVE_API URiveTextureObject : public URiveTexture, public FTickableGameObj
 
 #if WITH_EDITOR
 	void EditorBeginPlay();
+	void OnBeginPIE(bool bIsSimulating);
 	void OnEndPIE(bool bIsSimulating);
 	bool bHasBegunPlay = false;
```

Comment:
We can probably remove bHasBegunPlay at this point. We can also remove the small block of code in Tick that references it, and calls EditorBeginPlay

Reply:
Done.